4/26 (#103)
dolthub/weekly-updates: emails/2024-04-26.md

This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 

I've been real sick this week but that's not going to stop me from cranking out the weekly email. I've never heard of anyone catching a cold from an email.

### Dictionary Compression

One of the objections we hear to using Dolt is, "I can't possibly store all the history of my database in my database". [Neil](https://www.dolthub.com/team#neil) is on a mission to address this concern. 

First up is better compression for Dolt storage. Neil goes into detail [in his latest blog](https://www.dolthub.com/blog/2024-04-22-dolt-storage-dictionaries/) on how dictionary compression works and how it can compress Dolt databases by about 50% from current footprints. That's an exciting win especially when you consider the clone use case. Get a database from DoltHub twice as fast. Yes, please.

[![Compression Ratios](../images/compression-ratios.png)](https://www.dolthub.com/blog/2024-04-22-dolt-storage-dictionaries/)

> Next, Neil will follow up his [shallow clone work](https://www.dolthub.com/blog/2024-02-21-shallow-clone/) with a project we call history offloading where you can store the last N commits locally and have the rest of your history stored on [a remote](https://www.dolthub.com/blog/2024-04-25-why-remotes/), like DoltHub or DoltLab. 

Reviewer: ```suggestion
Next, Neil will follow up his [shallow clone work](https://www.dolthub.com/blog/2024-02-21-shallow-clone/) with a project we call "history offloading" where you can store the last N commits locally and have the rest of your history stored on [a remote](https://www.dolthub.com/blog/2024-04-25-why-remotes/), like DoltHub or DoltLab. 
```

> We think with these two changes, most users will feel comfortable storing the history of their database in their database using Dolt.

Reviewer 2: end of this sentence is hard to parse. How about:

With these two changes Dolt users can keep as much or as little of their history as they choose. How much history will you save? 1 month, 1 year, eternity? Up to you!

Author: I like it

### Replicate from Postgres to Doltgres

> We're still grinding away on [Doltgres](https://www.doltgres.com/), our Postgres-compatible version of Dolt. Doltgres got it's own [website](https://www.doltgres.com/) and [documentation](https://docs.doltgres.com/) this week to celebrate the launch of [replication from Postgres](https://www.dolthub.com/blog/2024-04-23-announcing-postgres-to-doltgres-replication/). 

Reviewer 3: ```suggestion
We're still grinding away on [Doltgres](https://www.doltgres.com/), our Postgres-compatible version of Dolt. Doltgres got its own [website](https://www.doltgres.com/) and [documentation](https://docs.doltgres.com/) this week to celebrate the launch of [replication from Postgres](https://www.dolthub.com/blog/2024-04-23-announcing-postgres-to-doltgres-replication/). 
```

> Replication is a great entry point into Dolt and Doltgres because you don't need to swap your production MySQL or Postgres benefit from Dolt or Doltgres version control features. It's purely additive. You get full history and diffs on your Dolt or Doltgres replica.

Reviewer: ```suggestion
Replication is a great entry point into Dolt and Doltgres because you don't need to swap your production MySQL or Postgres to benefit from Dolt or Doltgres version control features. It's purely additive. You get full history and diffs on your Dolt or Doltgres replica.
```

> Replication is a great entry point into Dolt and Doltgres because you don't need to swap your production MySQL or Postgres benefit from Dolt or Doltgres version control features. It's purely additive. You get full history and diffs on your Dolt or Doltgres replica.

Reviewer 3: ```suggestion
Replication is a great entry point into Dolt and Doltgres because you don't need to swap out your production MySQL or Postgres database to start benefiting from Dolt or Doltgres version control features. It's purely additive. You get full history and diffs on your Dolt or Doltgres replica.
```
